scene/2d/sprite_2d.h

```diff
@@ -89,6 +91,8 @@ class Sprite2D : public Node2D {
 	bool is_pixel_opaque(const Point2 &p_point) const;
 	bool is_editor_region_rect_draggable() const;
 
+	void set_dragging_to_resize_rect(bool p_dragging_to_resize_rect);
```

Comment:
This should be in the TOOLS_ENABLED part and likely prefixed with  `_edit`, as this only matters for the editor. Likely same for the `dragging_to_resize_rect` variable.